### DISCUSSION TITLE
Simplest vim motions in [r]eword commit message?

### DISCUSSION THREAD

**Discussion by antenando** [jesseduffield/lazygit#3445] [Ideas]
Hi! 1st of all thank you for this amazing project. Wouldn't change it for anything else.

Secondly, what do you think of very simple vim motions inside of the [r]eword commit msg panel? Like "dd" is the one I'm going for here, just to delete the line. But obviously other basic ones would be great too.

Thanks!

**Comment by mark2185** [comment]
There's no reason for that when you can open it up in your editor using `<c-o>` and `e`.

There are some emacs keybindings (which I don't remember) e.g. `<c-u>`, `<c-a>`, `<c-e>`, but the commit editor is supposed to be _simple_.

**Reply by thi-marques** [reply]
If there are some emacs keybinds it could have a simple vim mode as well since we can already navigate with `hjkl` outside the commit pop-up it would keep the consistency.

**Reply by mark2185** [reply]
Emacs keybindings are ubiquitous in shells and don't introduce modality, "simple vim mode" is anything but simple.

If you're writing such messages that you _need_ vim keybindings, open the commit editor in vim.
